src/doughnut/topping/mod.rs

```diff
@@ -1,8 +1,7 @@
-// Copyright 2022-2023 Futureverse Corporation Limited
+// Copyright 2022-2023 Futureverse Corporation Limited//!
```

Comment:
Something strange happened with this comment
```suggestion
//! Copyright 2022-2023 Futureverse Corporation Limited
```

Reply:
Should also change the copyright from 2023-2024